### ISSUE TITLE
[Bug]: Multiple Inactive Tabs CFRs are displayed after background foreground

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24187]
## Prerequisites
Have a couple of Inactive tabs

## Steps to reproduce
1. Open the tabs tray
2. Check if the Inactive tabs CFR is displayed
3. Put Fenix in background
4. Resume Fenix
5. Check the Inactive Tabs CFR
6. Tap the "Turn off in settings" link

### Expected behavior
Only one CFR should be displayed

### Actual behavior
Each time you perform a background/foreground a new CFR is displayed
Tapping the "Turn off in settings" link will redirect to the "Tabs" section still displaying a Inactive Tabs CFR

### Device information

* Android device: OnePlus A3 (Android 6.0.1)
* Fenix version: 
• Firefox Nightly 100.0a1
• Firefox 99.0.0 Beta 1

### Notes
► Video portrait

https://user-images.githubusercontent.com/51314259/157662456-92955de5-a9a7-4bb9-8b10-8ad9a7487d4c.mp4

► Video landscape

https://user-images.githubusercontent.com/51314259/157659829-86741ad5-b492-4bcd-85b8-57beb8ff2ff9.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-19789)

**Comment by SoftVision-LorandJanos** [comment]
Tested on the Nightly 100.0a1 (2022-03-10) build from the day this issue was logged on.
I reproduced the issue as described.
Tested on the latest Nightly 100.0a1 (2022-03-17) with the following results:
 - Tapping on the "Turn off in settings" link dismisses the CFR.
 - Putting Fenix on background/foreground doesn't dismiss the CFR.
 - Closing the app, reopening it and accessing the tabs tray, the CFR remains displayed.

**Comment by Alexandru2909** [comment]
Tagging UX to check if the behavior described by @SoftVision-LorandJanos in the previous comment is expected, so we can close the ticket.

**Comment by SoftVision-LorandJanos** [comment]
Tested this issue on the latest Nightly 109.0a1 (2022-11-23) build and I observed the following:
- the CFR remains on when the app is resumed from the background with the CFR active.
- tapping on the "Turn off in settings" link or in any point on the CFR will redirect to the "Tabs settings" section, and the CFR is dismissed and no longer displayed (This part seems fixed).
Tested on both portrait and landscape mode.
Device used: Oppo Find X5 (Android 12).
